hw/dv/data/dsim/dsim.hjson

```diff
@@ -14,25 +14,24 @@
                // UVM
                "+incdir+{UVM_HOME}/src",
                "{UVM_HOME}/src/uvm_pkg.sv",
-               "+acc+rwb -sv_lib {DSIM_HOME}/lib/libuvm_dpi.so",
-
+               "-sv_lib {DSIM_HOME}/lib/libuvm_dpi.so",
                // Add dpi/vpi include path.
-               "-c_opts -I{DSIM_HOME}/include",
-
+               "-c-opts -I{DSIM_HOME}/include",
                "-timescale 1ns/1ps",
                "-f {sv_flist}",
                "+incdir+{build_dir}",
                // Suppress following DSim errors and warnings:
                //   EnumMustBePositive - UVM 1.2 violates this
-               "-suppress EnumMustBePositive",
+               "-suppress EnumMustBePositive"
               ]
 
   run_opts:   [
                "-work {build_dir}/dsim_out",
                "-image image",
                // UVM DPI
-               "+acc+rwb -sv_lib {DSIM_HOME}/lib/libuvm_dpi.so",
-               "+ntb_random_seed={seed}",
+               "-sv_lib {UVM_HOME}/src/dpi/libuvm_dpi.so",
+               "-sv_seed {seed}",
+               "-linebuf",
```

Comment:
It will be useful to know what a tool switch does with a comment, especially if it is not obvious enough.

Reply:
Certainly. Let me know what you think of this:
```
// tell DSim to write line-buffered std output (lines will be written in proper order)
"-linebuf",
```
The switch is optional but recommended if you ever look at DSim's std output in a terminal. Without this switch the order in which lines are written to the terminal may be confusing. If you can't think of a case where this would be useful I can remove it. There is no penalty for leaving it in.

Please let me know if the comment is acceptable and I'll make another commit.

Reply:
Yes, that looks good, thanks.

Reply:
Fixed here: 3634825